### ISSUE TITLE
Document new Note renderings `> **Note**`, `> **Warning**`, etc.

### ISSUE THREAD

**Issue by aisgbnok** [github/docs#18164]
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

- [Basic writing and formatting syntax](https://docs.github.com/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-text)
   - Under the "Quoting text" section OR
   - Under its own section
- [Working with advanced formatting](https://docs.github.com/get-started/writing-on-github/working-with-advanced-formatting) having its own page?

### What part(s) of the article would you like to see updated?

I just noticed that using `> **Note**` will generate a rendered/highlighted Note with a little info icon.

> **Note**

I use this syntax in some markdown files, and it just started rendering this way.  I don't believe this is documented anywhere. I'm not entirely sure the best location to put this, you can see my suggestions above? I wouldn't mind writing and opening a PR documenting this, but I first would love some feedback as to where this would be best placed. Thanks!

### Additional information

I love these new little improvements that are being done across the site, similar to #17312 with color rendering `#3fb950`, [adding LaTeX rendering](https://github.blog/changelog/2022-05-19-render-mathematical-expressions-in-markdown/), and now this new Note rendering. 

I wonder if there are other instances for something like Info, warning, error, etc.

> **Note**
> Yes

> **Error**
> No

> **Warning**
> Yes

> **Info**
> No

> **Success**
> No

Cool, look at that, just found another one. I'd love to see how these develop and help document them.

**Comment by aisgbnok** [comment]
https://github.com/github/feedback/discussions/16925#discussion-4085374

**Comment by janiceilene** [comment]
@aisgbnok Thanks so much for opening a PR! I'll get this triaged for review ⚡

**Comment by huyz** [comment]
Isn't this still a "beta feature" "subject to change"?

**Comment by aisgbnok** [comment]
@huyz, yes, to my knowledge this is still [being tested as a beta feature.](https://github.com/orgs/github-community/discussions/16925#discussion-4085374)

Once GitHub finalizes the syntax, design, etc. they will most likely document it. If they choose to leave it to public contribution, then a PR can be opened in this repo. This issue can be closed once the documentation is published, or if it's deemed unnecessary, etc.

**Comment by janiceilene** [comment]
👋 @aisgbnok I've spoken with the team and we're not ready to add this to the docs yet, since it's so early in the beta. I'm going to close this now 💛 

Thank you so much for your interest and passion in improving the GitHub Docs ⚡
